examples/xgboost-quickstart/xgboost_quickstart/server_app.py

```diff
@@ -0,0 +1,50 @@
+"""xgboost_quickstart: A Flower / XGBoost app."""
+
+from typing import Dict
+
+from flwr.common import Context
+from flwr.server import ServerApp, ServerAppComponents, ServerConfig
+from flwr.server.strategy import FedXgbBagging
+
+
+def evaluate_metrics_aggregation(eval_metrics):
+    """Return an aggregated metric (AUC) for evaluation."""
+    total_num = sum([num for num, _ in eval_metrics])
+    auc_aggregated = (
+        sum([metrics["AUC"] * num for num, metrics in eval_metrics]) / total_num
+    )
+    metrics_aggregated = {"AUC": auc_aggregated}
+    return metrics_aggregated
+
+
+def config_func(rnd: int) -> Dict[str, str]:
+    """Return a configuration with global epochs."""
+    config = {
+        "global_round": str(rnd),
+    }
+    return config
+
+
+def server_fn(context: Context):
+    # Read from config
+    num_rounds = context.run_config["num-server-rounds"]
+    fraction_fit = context.run_config["fraction-fit"]
+    fraction_evaluate = context.run_config["fraction-evaluate"]
+
+    # Define strategy
+    strategy = FedXgbBagging(
```

Comment:
@yan-gao-GY , should we pass the `initial_parameters` just like discussed for the xgboost template?

Reply:
@jafermarq We decided not passing `initial_parameters`, and just remove `get_parameters` in `client_app`.